insights/parsers/sap_dev_trace_files.py

```diff
@@ -0,0 +1,179 @@
+"""
+from insights import parser, LogFileOutput
+from insights.specs import Specs
+from insights.parsers import ParseException
+from insights.core.filters import add_filter
+
+add_filter(Specs.sap_dev_disp, 'trc file:')  # keep the header line by default
+add_filter(Specs.sap_dev_rd, 'trc file:')  # keep the header line by default
+
+
+@parser(Specs.sap_dev_disp)
+class SapDevDisp(LogFileOutput):
+    """
+    This class reads the SAP trace files ``/usr/sap/SID/SNAME/work/dev_disp``
+
+    Sample content of the file::
+
+        ---------------------------------------------------
+        trc file: "dev_disp", trc level: 1, release: "745"
+        ---------------------------------------------------
+
+        Sun Aug 18 17:17:45 2019
+        TRACE FILE TRUNCATED ( pid =    16955 )
+
+        sysno      00
+        sid        RH1
+        systemid   390 (AMD/Intel x86_64 with Linux)
+        relno      7450
+        patchlevel 0
+        patchno    100
+        intno      20151301
+        make       multithreaded, Unicode, 64 bit, optimized
+        profile    /usr/sap/RH1/SYS/profile/RH1_D00_vm37-39
+        pid        16955
+
+        Sun Aug 18 17:17:45 2019
+        kernel runs with dp version 3000(ext=117000) (@(#) DPLIB-INT-VERSION-0+3000-UC)
+        length of sys_adm_ext is 500 bytes
+
+        Scheduler info
+        --------------
+        WP info
+          #dia     = 10
+          #btc     = 3
+          #standby = 0
+          #max     = 21
+        General Scheduler info
+          preemptionInfo.isActive           = true
+          preemptionInfo.timeslice          = 500
+          preemptionInfo.checkLoad          = true
+        Prio Class High
+          maxRuntime[RQ_Q_PRIO_HIGH]     = 600 sec
+          maxRuntimeHalf[RQ_Q_PRIO_HIGH] = 300 sec
+        Running requests[RQ_Q_PRIO_NORMAL] = 0
+        Running requests[RQ_Q_PRIO_LOW] = 0
+
+        *** WARNING => DpHdlDeadWp: wp_adm slot for W7 has no pid [dpxxwp.c     1353]
+        DpSkipSnapshot: last snapshot created at Sun Aug 18 17:15:25 2019, skip new snapshot
+        *** WARNING => DpHdlDeadWp: wp_adm slot for W8 has no pid [dpxxwp.c     1353]
+        DpSkipSnapshot: last snapshot created at Sun Aug 18 17:15:25 2019, skip new snapshot
+        *** WARNING => DpHdlDeadWp: wp_adm slot for W9 has no pid [dpxxwp.c     1353]
+
+        Sun Aug 18 17:17:45 2019
+        DpSkipSnapshot: last snapshot created at Sun Aug 18 17:17:45 2019, skip new snapshot
+        DpCheckSapcontrolProcess: sapcontrol with pid 1479 terminated
+        *** WARNING => DpRequestProcessingCheck: potential request processing problem detected (14. check) [dpxxwp.c     4633]
+
+    Examples:
+        >>> type(dev_disp)
+        <class 'insights.parsers.sap_dev_trace_files.SapDevDisp'>
+        >>> len(dev_disp.get("WARNING"))
+        4
+    """
+
+    def get_after(self, *args, **kwargs):
+        raise ParseException("get_after() is not supported by this Parser.")
```

Comment:
@xiangce would you add docstring to this method to replace the default docstring which is incorrect for your method implementation?

Reply:
@bfahr - Done. Please check it again.

Reply:
@bfahr - May I have your review on this PR, Thanks.